src/editorwidget/plugin.ts

```diff
@@ -80,7 +84,9 @@ const cmdIds = {
   matchBrackets: 'editor:match-brackets',
   vimMode: 'editor:vim-mode',
   closeAll: 'editor:close-all',
-  changeTheme: 'editor:change-theme'
+  changeTheme: 'editor:change-theme',
+  startConsole: 'editor:start-console',
```

Comment:
Are we using "start" or "create" in the UI/commands. Let's be consistent as it makes it easier for users to remember the names of commands.